app/src/main/java/com/example/easysurvey/MainActivity.kt

```diff
@@ -23,40 +28,78 @@ class MainActivity : ComponentActivity() {
             EasySurveyTheme {
                 // A surface container using the 'background' color from the theme
                 Surface(modifier = Modifier.fillMaxSize(), color = MaterialTheme.colors.background) {
-                    HomepageButtons()
+                    homepageButtons()
```

Comment:
changed function to camelCase as per the Kotlin coding conventions: https://kotlinlang.org/docs/coding-conventions.html#function-names